specification/attributes/column_naming_convention.md

```diff
@@ -0,0 +1,34 @@
+# Column naming convention
+
+Column IDs provided in billing datasets following a consistent naming convention reduces friction for FinOps
```

Comment:
nit: I'm getting hung up on "billing" datasets. Technically, the columns we define will overlap with data that isn't directly related to the bill, like recommendations. Recommendations are about cost, not bills. Should we talk about "cost data" or "cost details" or something? Not a big deal. I just keep thinking about the related datasets that we'll overlap with and not all of them are related to getting billed and paying invoices.

Reply:
Sorry what's the recommendation here. Instead of saying "billing datasets" use ....?

Reply:
Updated to say cost data. Not sure about recommendations but saying cost data here seems like a fine change.